STM32G431KBTx/makefile

```diff
@@ -0,0 +1,204 @@
+######################################
+# target
+######################################
+TARGET = nucleo
```

Comment:
I'd prefer if the TARGET name were a bit more descriptive, e.g., `nucleo431`. Not a huge thing tho.